src/app/calc/eval/clickWand.ts

```diff
@@ -242,9 +242,8 @@ export function clickWand(
 
     if (
       !fireUntilReload ||
-      (endOnRefresh &&
-        calledActions!.filter((a) => a.action.id === 'RESET').length > 0) ||
-      calledActions!.length === 0
+      calledActions!.length === 0 ||
+      (endOnRefresh && calledActions!.some((a) => a.action.id === 'RESET'))
```

Comment:
This isn't related to the undo/redo bit, changed it while trying to fix the add trigger bug and it feels slightly better :)